src/sourmash/commands.py

```diff
@@ -477,7 +497,8 @@ def search(args):
         notify("** reporting only one match because --best-only was set")
 
     if args.output:
-        fieldnames = ['similarity', 'name', 'filename', 'md5']
+        fieldnames = ['similarity', 'name', 'filename', 'md5',
+                      'query_filename', 'query_name', 'query_md5']
```

Comment:
It would be nice to modify `similarity` to `containment` / `max_containment` for csv output. Thoughts?

Reply:
semantic versioning prevents us from removing the `similarity` header before 5.0. we could add new columns, I 'spose. I don't like the idea that column headers **change** depending on command line arguments, though. Not sure how to think about it.

Reply:
(suggest we punt this to a new issue and discuss it there.)

Reply:
punted to https://github.com/dib-lab/sourmash/issues/1390